components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -1086,6 +1087,17 @@ - (void)fhv_contentOffsetDidChange {
   [self fhv_updateLayout];
 }
 
+- (CGFloat)additionalSpaceNeededForDeviceTopSafeAreaInset {
+  return [self isWithinDeviceTopSafeAreaInsets] ? 0 : MDCDeviceTopSafeAreaInset();
+}
+
+- (BOOL)isWithinDeviceTopSafeAreaInsets {
+  UIWindow *keyWindow = [UIApplication mdc_safeSharedApplication].keyWindow;
```

Comment:
Is it possible to have an approach that also works in extensions? I believe sharedApplication is unavailable in extensions.